api/api.py

```diff
@@ -235,3 +235,34 @@ def default_registry(registry: str) -> None:
   (Note: this logic is currently crude, on the assumption that development image names are ephemeral and unimportant. `Please let us know <https://github.com/windmilleng/tilt/issues>`_ if they don't suit you!)
   """
   pass
+
+class CustomBuild:
+  """An image that was created with ``custom_build``"""
+  def add_fast_build() -> 'CustomBuild':
+    """Returns a FastBuild that is associated with the image that was built from a ``custom_build``."""
+    pass
+
+def custom_build(ref: str, command: str, deps: List[str], disable_push: bool = False) -> CustomBuild:
+  """Provide a custom command that will build an image.
+
+  Returns an object which can be used to create a FastBuild.
+
+  It will raise an error if the specified ref is not published in the registry with the tag that is provided via the ``$TAG`` environment variable.
```

Comment:
```suggestion
  It will raise an error if the specified ref is not published in the registry with the name+tag that is provided via the ``$EXPECTED_REF`` environment variable.
```